### DISCUSSION TITLE
Page Unresponsive when converting Google Keep

### DISCUSSION THREAD

**Discussion by zylstra** [standardnotes/forum#2739] [General]
**Describe the issue**
I tried to convert 469 files (with a total size of about 100 MB, mainly images) from my Google Keep folder with the [conversion tool](https://dashboard.standardnotes.org/tools) but the Page Unresponsive dialogue box keeps popping up in Chrome.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome
 - Version latest

**Comment by zylstra** [comment]
I tried Firefox private window and got a little further:  I saw the popup that told me to send my console errors to help@standardnotes.org; so I did.

**Comment by JaspalSuri** [comment]
Hi @zylstra, thank you for reaching out to us. A report has been created for the dev team to review. 🙂
